spec/importers/californica_importer_spec.rb

```diff
@@ -26,10 +26,10 @@
       expect(importer.parser).to have_received(:reindex_collections)
     end
 
-    it 'builds IIIF manifests' do
-      importer.finalize_import
-      expect(importer.parser).to have_received(:build_iiif_manifests)
-    end
+    # it 'builds IIIF manifests' do
```

Comment:
same question as above: If we disable via a comment, should we also include an explanation of why it is commented?